hello/hello_patch.py

```diff
@@ -0,0 +1,149 @@
+import asyncio
+import logging
+import sys
+from dataclasses import dataclass
+from datetime import timedelta
+
+from temporalio import activity, exceptions, workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+
+# While we could use multiple parameters in the activity, Temporal strongly
+# encourages using a single dataclass instead which can have fields added to it
+# in a backwards-compatible way.
+@dataclass
+class ComposeGreetingInput:
+    greeting: str
+    name: str
+
+
+# Basic activity that logs and does string concatenation
+@activity.defn
+async def compose_greeting(input: ComposeGreetingInput) -> str:
+    activity.logger.info("Running activity with parameter %s" % input)
+    return f"{input.greeting}, {input.name}!"
+
+
+# V1 of patch-workflow
+@workflow.defn(name="patch-workflow")
+class MyWorkflow:
+    @workflow.run
+    async def run(self, name: str) -> str:
+        print(f"Running patch-workflow with parameter %s" % name)
+        workflow.logger.info("Running patch-workflow with parameter %s" % name)
+        greeting = await workflow.execute_activity(
+            compose_greeting,
+            ComposeGreetingInput("Hello", name),
+            start_to_close_timeout=timedelta(seconds=70),
+        )
+        return greeting
+
+
+# V2 of patch-workflow using patched where we have changed newly started
+# workflow behavior without changing the behavior of currently running workflows
+@workflow.defn(name="patch-workflow")
+class MyWorkflowPatched:
+    @workflow.run
+    async def run(self, name: str) -> str:
+        print(f"Running patch-workflow with parameter %s" % name)
```

Comment:
Should not use print from inside a workflow. Can use workflow logger if you want.

Reply:
ok ill swap that